vscode-dotnet-runtime-library/src/LocalMemoryCacheSingleton.ts

```diff
@@ -110,6 +112,11 @@ export class LocalMemoryCacheSingleton
             {
                 return undefined;
             }
+            else if (k === 'env')
+            {
+                return `${minimizeEnvironment(v)}-${TelemetryUtilities.HashData(JSON.stringify(v))}`; // Each command with a unique env must be cached uniquely -- it's helpful to see what the important vars are in the log.

```

Comment:
Does this hash values separately or together? Meaning would you be able to differentiate between:
{ "a": "b", "c": "d" } and {"c": "d", "a": "b"}
?

Reply:
Those would be differentiated. It hashes the entire object. Interesting to note that if the ordering of the keys in the object are different that may make it get cached separately/twice, but I don't think that that would happen.

Reply:
Ah, that's a good point ;) That's why I included the minimized environment at the front so we can see the things we need. The included hash is to make sure we don't cache things that are actually different environments because we minimized away all of the differences. The approach you mention would be helpful in differentiating variables that we don't need to collect but might have an unintended impact, however. What do you think? 

Thanks for taking a look.